content/source/docs/cloud/sentinel/sentinel-tf-012.html.md

```diff
@@ -170,6 +184,35 @@ namespace](./import/tfplan.html#namespace-resource-diff)
 to validate whether or not a value is unknown before looking for it in
 `applied`.
 
+### Changes Affecting Resources Being Destroyed but not Re-created
+
+In Terraform 0.11, when a resource is being destroyed but not re-created, it's
+[`diff`](./import/tfplan.html#value-diff) value in the `tfplan` import is empty.
+In Terraform 0.12, however, the `diff` value does has data. For both versions of
```

Comment:
"value does ~has~have data." or "value ~does~ has data."

Reply:
thanks for catching this.  Fixed.